apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -5448,7 +5448,8 @@
     "drawer": {
       "title": "Choose an image",
       "options": {
-        "uploadFromPhone": "Upload from my phone"
+        "uploadFromPhone": "Upload from my phone",
+        "selectFromNFTGallery": "Select from NFT gallery"
```

Comment:
can you use final wording "Choose from NFT gallery" please?  (it changed yesterday I think)

Reply:
Done